compiled/web_bindings/plonk_wasm.js

```diff
@@ -711,117 +711,117 @@ export function caml_pasta_fq_poseidon_block_cipher(state) {
 }
 
 /**
-* @param {WasmFpGateVector} gates
+* @param {WasmFqGateVector} gates
 * @param {number} public_
 * @param {number} prev_challenges
-* @param {WasmFpSrs} srs
-* @returns {WasmPastaFpPlonkIndex}
+* @param {WasmFqSrs} srs
+* @returns {WasmPastaFqPlonkIndex}
```

Comment:
I don't really have a lot of context on this change, but why are Fp and Fq here being swapped?

Reply:
This is auto-generated, it comes from the output of `npm run bindings`, which runs the tool `wasm-pack` we don't have control on. I wish also it would not always change and be a bit more deterministic.

Reply:
I note Fp and Fq are still both generated, just in different positions I guess. 👍🏼